src/style.css

```diff
@@ -1,3 +1,65 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+
+  --active-menu: #00acdc;
```

Comment:
Create a variable with root selector

Reply:
I don't understand, what should I need to change here?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties
![image](https://github.com/mate-academy/layout_moyo-header/assets/96265844/92d9a2a2-73c8-4d17-9678-a2f10e3abfa6)